cli/src/main/kotlin/commands/DownloaderCommand.kt

```diff
@@ -198,7 +218,14 @@ class DownloaderCommand : CliktCommand(name = "download", help = "Fetch source c
                     }
                 }
 
-                val packageDownloadDirs = packages.associateWith { outputDir.resolve(it.id.toPath()) }
+                val packageDownloadDirs = if (licenseClassificationsFile.isFile) {
+                    val licenseCategorizations =
+                        licenseClassificationsFile.readValue<LicenseClassifications>().orEmpty().categorizations
+
+                    packages.filter { isLicenseCategoryIncluded(it, ortResult, licenseCategorizations) }
```

Comment:
I'm thinking about the function name again. Shouldn't it rather be something like `isLicenseInIncludedCategory` or `isLicenseIncludedViaCategory` or `isLicenseIncludedByCategory`? Because we're not checking here whether the license category itself is included.